core/CoreDatabaseManager.java

```diff
@@ -17,51 +17,54 @@
  * under the License.
  */
 
-package grakn.client.rpc;
+package grakn.client.core;
 
-import grakn.client.GraknClient;
-import grakn.client.common.exception.GraknClientException;
-import grakn.protocol.DatabaseProto;
+import grakn.client.api.database.Database;
+import grakn.client.api.database.DatabaseManager;
+import grakn.client.common.GraknClientException;
+import grakn.client.common.Proto;
 import grakn.protocol.GraknGrpc;
 
 import java.util.List;
 import java.util.stream.Collectors;
 
-import static grakn.client.common.exception.ErrorMessage.Client.DB_DOES_NOT_EXIST;
-import static grakn.client.common.exception.ErrorMessage.Client.MISSING_DB_NAME;
+import static grakn.client.common.ErrorMessage.Client.DB_DOES_NOT_EXIST;
+import static grakn.client.common.ErrorMessage.Client.MISSING_DB_NAME;
 
-public class DatabaseManagerRPC implements GraknClient.DatabaseManager {
-    private final ClientRPC client;
+public class CoreDatabaseManager implements DatabaseManager {
+    private final CoreClient client;
     private final GraknGrpc.GraknBlockingStub blockingGrpcStub;
 
-    public DatabaseManagerRPC(ClientRPC client) {
+    public CoreDatabaseManager(CoreClient client) {
         this.client = client;
         blockingGrpcStub = GraknGrpc.newBlockingStub(client.channel());
     }
 
     @Override
-    public boolean contains(String name) {
-        return client.call(() -> blockingGrpcStub.databaseContains(DatabaseProto.Database.Contains.Req.newBuilder().setName(nonNull(name)).build()).getContains());
+    public boolean contains(String n) {
```

Comment:
hmmm 
```suggestion
    public boolean contains(String name) {
```

(Also applies to `create`, see below)